server/netty-server/cats/src/test/scala/sttp/tapir/server/netty/cats/NettyCatsServerTest.scala

```diff
@@ -20,7 +20,8 @@ class NettyCatsServerTest extends TestSuite with EitherValues {
           val interpreter = new NettyCatsTestServerInterpreter(eventLoopGroup, dispatcher)
           val createServerTest = new DefaultCreateServerTest(backend, interpreter)
 
-          val tests = new AllServerTests(createServerTest, interpreter, backend, staticContent = false, multipart = false).tests()
+          val tests = new AllServerTests(createServerTest, interpreter, backend, staticContent = false, multipart = false).tests() ++
+            new ServerMultipartTests(createServerTest, partOtherHeaderSupport = false).tests()
```

Comment:
can we support the additional part headers? seems this might be a matter of adding the headers when serialising the part?

Reply:
As far I as check there is no way to get those headers while parsing

Reply:
hm it would be weird if the data just got lost

Reply:
I will check it once more

Reply:
After some more digging I cannot see any way to get parts headers from netty - they are not included in overall request headers nor in `HttpDataType's` used in `getParts` method

Reply:
yeah it seems so; so we'll have to leave this as is, thanks for checking